### PR TITLE
Added Caravaneer

### PR DESCRIPTION
**Pull request by cadowtin**

### CODE REVIEW

**Comment by cadowtin** [comment]
it's very broken and I am very lost as to why

File: subclass/PosTavern; Artificer Specialist - Caravaneer.json [hunk 1]
```diff
@@ -0,0 +1,648 @@
...
+				"number": 25,
+				"condition": "(increases to 50 ft. at <b>9th level</b>)"
+			}
+		},
+		"str": 18,
+		"dex": 7,
+		"con": 20,
+		"int": 0,
+		"wis": 0,
+		"cha": 0,
```
Comment (revilowaldow) [inline subclass/PosTavern; Artificer Specialist - Caravaneer.json:49]:
```suggestion
		"cha": 0,
		"passive": null,
```

File: subclass/PosTavern; Artificer Specialist - Caravaneer.json [hunk 2]
```diff
@@ -0,0 +1,648 @@
...
+				"Once per round, if an effect that deals fire damage occurs to the fireplace, the arcane caravan regains {@dice 2d4 + 2} hit points. If its hit points are reduced to zero, all creatures and objects within the extradimensional dwelling are expelled through the chimney and fall prone into the nearest open spaces. No creatures can enter or recall the arcane caravan until it is revived. You can spend 1 hour working with mason’s tools and expend a spell slot of 1st level or higher to revive it with all of its hit points restored.",
+				"Using {@item mason's tools|PHB}, you can spend 8 hours to construct a new arcane caravan. If you already have an arcane caravan from this feature, the first one immediately perishes."
+			]
+		},
+		{
+			"name": "Sentient Host",
+			"source": "AS:C",
+			"page": 3,
+			"className": "Artificer",
+			"classSource": "TCE",
```
Comment (revilowaldow) [inline subclass/PosTavern; Artificer Specialist - Caravaneer.json:270]:
```suggestion
			"classSource": "TCE",
			"subclassShortName": "Caravaneer",
			"subclassSource": "AS:C",
```

**Review by revilowaldow — CHANGES_REQUESTED** [review]

**Comment by cadowtin** [comment]
Okay it works now!

**Comment by cadowtin** [comment]
Thank you for the help!

**Review by TheGiddyLimit — APPROVED** [review]

**Pull request merged**